gubernator/github/handlers.py

```diff
@@ -21,6 +21,7 @@
 import traceback
 
 import webapp2
+from webapp2_extras import security
 
 from google.appengine.datastore.datastore_query import Cursor
```

Comment:
Please import modules not classes https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports